lms/djangoapps/verify_student/models.py

```diff
@@ -569,15 +552,15 @@ def display_off(cls, user_id):
         property to false, so the notification banner can be switched off.
         """
         user = User.objects.get(id=user_id)
-        cls.objects.filter(user=user, status="denied").exclude(window=None).update(display=False)
+        cls.objects.filter(user=user, status="denied").update(display=False)
```

Comment:
Please also add a comment for the `display` field on the model indicating that it has been deprecated (but please don't delete it)